app/models/concerns/course/levels_concern.rb

```diff
@@ -41,4 +41,35 @@ def compute_level_number(experience_points)
     level = compute_level(experience_points)
     level ? level.level_number : 0
   end
+
+  # Returns the current level's experience point (exp) threshold for
+  # for a given number of experience_points.
```

Comment:
Given the following Course and levels: 
Level 1 - 100, Level 2 - 300, Level 3  - 500

Say Student has 250 EXP, he is level 1, and his current_level_threshold is 100 (threshold of level 1). Will need this to calculate the difference between the threshold of the current level and next level to determine how far the CourseUser has progressed in that particular level. 

Can suggest to reword the explanation a little, to `Returns the experience point's threshold for the current level, which is determined by the experience_points parameter`. Beyond that, the other solution is to rename the method, but I do not have a better idea >_<